etcdserver/etcdserverpb/rpc.proto

```diff
@@ -430,6 +437,15 @@ message MemberListResponse {
   repeated Member members = 2;
 }
 
+message MemberStatusRequest {
+  uint64 ID = 1;
+}
+
+message MemberStatusResponse{
+  ResponseHeader header = 1;
+  MemberStatus status = 2;
```

Comment:
just put string version = 2; here? This is already status response. (we might have raftStatus = 3; dbStatus = 4 in the future) Embedding another status does not make sense to me. Or you have other plan?